legacy/preferences/src/main/java/app/k9mail/legacy/preferences/GeneralSettingsChangeListener.kt

```diff
@@ -0,0 +1,5 @@
+package app.k9mail.legacy.preferences
+
+fun interface GeneralSettingsChangeListener {
```

Comment:
This could be more generic like `SettingsChangeListener` and `onSettingsChanged()`